minesweeper.py

```diff
@@ -117,13 +117,14 @@ def update(index):
                 )
 
     def timer(self):
-        if self.game_status == GameStatus.in_progress:
-            statusbar_time.config(text=self.game_time.strftime("%M:%S"))
-            self.game_time += datetime.timedelta(seconds=1)
-            root.after(1000, self.timer)
-        elif self.game_status == GameStatus.game_won:
-            with open("high_scores.json", "w") as high_scores:
-                json.dump(self.game_time.strftime("%M:%S"), high_scores)
+        if self == current_game:
```

Comment:
This is slightly better style. I can later explain why.

```suggestion
        if self is current_game:
```